### DISCUSSION TITLE
Hot reloading + rechecking on .eslintrc.js changes

### DISCUSSION THREAD

**Discussion by iliubinskii** [quasarframework/quasar#13672] [Ideas / Proposals]
Quasar dev server reloads on quasar.config.js changes, but not on .eslintrc.js changes.

**Comment by suleimanas** [comment]
https://webpack.js.org/configuration/dev-server/#watch

https://vitejs.dev/config/#server-watch

**Comment by iliubinskii** [comment]
Hi, thank you for the hint.

Actually, dev server reloads on .eslintrc edit even without watch or watchFiles.
The problem is that this reload does not trigger new eslint check.

Adding watch or watchFiles option does not change anything:
Dev server reloads but it already reloaded before.
Reload does not trigger new eslint check as it did not before.

**Comment by iliubinskii** [comment]
I have changed the subject to "Hot reloading + rechecking on .eslintrc.js changes" (added "+ rechecking"), because the goal is not just to reload, but also to re-check.
